core/lib/state/src/utils.rs

```diff
@@ -0,0 +1,21 @@
+use std::path::PathBuf;
+
+use anyhow::Context;
+use zksync_storage::RocksDB;
+
+use crate::StateKeeperColumnFamily;
+
+/// Opens a RocksDB database with the given path and family descriptors used by state keeper.
+///
+/// # Errors
+///
+/// Propagates RocksDB I/O errors.
+pub async fn open_state_keeper_rocksdb(
```

Comment:
Bikeshedding: IMO, this change can be avoided; you can still retain `RocksdbStorage::builder()` method as a way to initialize the storage + convert the builder to `RocksDB<StateKeeperColumnFamily>` if required.

Reply:
Yeah, I managed to return RocksDB-related code to almost what it used to be before this PR.

Also, I got rid of all `From` implementations as they didn't fit the official `From` [guidelines](https://doc.rust-lang.org/std/convert/trait.From.html). Props to @popzxc for his Slack post about this the other day